index.html

```diff
@@ -0,0 +1,35 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Raf/Ronny Project</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <header>
+    <nav id="navbar"> 
+
+      <a href="#welcome-section">About Us</a>
+      <a href="#projects">Projects</a>
+      <a href="https://github.com/roncortez" target="_blank" id="profile-link">Ronny's Github Profile</a>  
+      <a href="https://github.com/3nityGitHub">Raf's Github Profile</a>
+   
+    </nav>     
+  </header>
+
+
+  <body>
+    
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <script  src="js/index.js"></script>
+    <section id="welcome-section">
+      <h1>Hello! This is Raf&Ronny's project portfolio</h1>
+      <p>We are software developers at Microverse..</p>
```

Comment:
Please kindly wrap this line of code in a <div> container, this will improve readability. [click here to read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div)

Reply:
- [ ] Kindly make sure you provide a valid reason why changes are required